src/app/AppLayout/AppLayout.tsx

```diff
@@ -54,7 +54,8 @@ const AppLayout: React.FunctionComponent<IAppLayout> = ({children}) => {
   };
 
   const Navigation = (
-    <Nav id="nav-primary-simple" theme="dark" variant="default">
+    <Nav id="nav-primary-simple" theme="dark" variant="default" 
+        onSelect={(selected) => {if(isMobileView) setIsNavOpenMobile(false)}}>
```

Comment:
That's kind of an awkward one-liner with the if-statement in there. Could you extract it out to a named function?